config/ftbquests/normal/chapters/08d906a0/147b1b5f.snbt

```diff
@@ -25,7 +25,87 @@
 				}
 			}
 		}],
+		consume_items: true,
```

Comment:
I like the idea of "upgrading" crook,
Did you tested this in game? Is it working properly?

Reply:
Yes, I have tested it, and it works.